src/app/registration/contact-form/contact-form.component.html

```diff
@@ -103,25 +103,36 @@ <h3>
           <label class="mb-2" for="phone">
             Phone Number:
           </label>
-          <input 
-            class="form-control mb-1" 
-            type="tel" 
-            formControlName="phone" 
-            id="phone" 
-            name="phone" 
-            placeholder="111-222-3333"
-            maxlength="12"
-            appPhoneFormat 
-            />
-          <div *ngIf="myForm.errors?.['phoneRequired']">
-            <small *ngIf="myForm.errors?.['phoneRequired']" class="text-danger">*Your Phone Number is Required</small>
+        <ngx-intl-tel-input
```

Comment:
Browser warnings are present:
```
 It looks like you're using ngModel on the same form field as formControlName.
  Support for using the ngModel input property and ngModelChange event with
  reactive form directives has been deprecated in Angular v6 and will be removed
  in a future version of Angular.
```

May need to refactor to clear the deprecation warnings.

Reply:
Discussed at standup, I will create a new ticket to revise the other fields of the form to only use reactive forms. I have updated the logic for the phone input.

Addressed in: https://github.com/bcgov/parks-reso-public/pull/304/commits/1a599013cd01abb93ec6705be0b8577b355eb442